conductr_cli/sandbox_run.py

```diff
@@ -186,15 +189,27 @@ def stop_nodes(args, running_containers):
 def wait_for_start(args):
     if not args.no_wait:
         log = logging.getLogger(__name__)
+        print('Waiting for ConductR to start', end='', flush=True)
```

Comment:
`print` with `end='', flush=True` is necessary in order to print on the same line